tests/test_service.py

```diff
@@ -108,3 +108,35 @@ def test_service_with_replacement_docker():
   </service>
 </services>"""
     )
+
+
+def test_service_with_multi_flavor_docker():
```

Comment:
possibly worth adding a test to check if `_multibuild` is correct, not just for `_service`.

Reply:
tests added.